hadoop-ozone/dist/src/main/compose/ozone/test.sh

```diff
@@ -58,19 +58,16 @@ execute_robot_test scm -v SCHEME:o3fs -v BUCKET_TYPE:link -N ozonefs-simple-o3fs
 export OZONE_KEEP_RESULTS=true
 stop_docker_env
```

Comment:
Since there is no OBS supported in fs semantics, its not required to stop/restart cluster scenario.

Please remove lines starting from [ozone/test.sh_L54](https://github.com/apache/ozone/blob/master/hadoop-ozone/dist/src/main/compose/ozone/test.sh#L54) to [ozone/test.sh_L64](https://github.com/apache/ozone/blob/master/hadoop-ozone/dist/src/main/compose/ozone/test.sh#L64).

Reply:
Thank you for pointing this out, I have removed this.